subtests/docker_cli/kill_stopped/kill_utils.py

```diff
@@ -80,7 +80,7 @@ def _init_container_normal(self, name):
         subargs.append("bash")
```

Comment:
Yay! I've been wondering but hadn't looked close enough yet (to see if they could be combined). Thanks! Glad to know it's not forgotten.